addons/website/views/snippets/s_cards_soft.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_cards_soft" name="Cards Soft">
+    <section class="s_cards_soft o_cc o_cc1 pt32 pb32 o_colored_level" data-vxml="001" data-vcss="001">
```

Comment:
Why do we need versioning if this PR introduces this snippet ? IMO we can get rid of `000.scss` and it's record at the end of this file

Also, `o_colored_level` does not seem useful here.


```suggestion
    <section class="s_cards_soft o_cc o_cc1 pt32 pb32">
```
```suggestion
    <section class="s_cards_soft o_cc o_cc1 pt32 pb32 o_colored_level" data-vxml="001" data-vcss="001">
```